net-libs/libblkmaker/libblkmaker-0.5.3.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A library implementing getblocktemplate"
+DESCRIPTION="libblkmaker provides an implementation of getblocktemplate, the\
+ current Bitcoin mining protocol."
+HOMEPAGE="https://github.com/bitcoin/libblkmaker"
+COPYRIGHT="Luke Dashjr"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/libblkmaker/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="549193640d4cc7022725754833d9d8a7639c41e1a7d2c5f2588be65857cc0970"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+if [[ "$targetArchitecture" == "x86_gcc2" && -z "$secondaryArchSuffix" ]]; then
+	PATCHES="libblkmaker-$portVersion.patchset"
+fi
+
+PROVIDES="
+	libblkmaker$secondaryArchSuffix = $portVersion
+	lib:libblkmaker_0.1$secondaryArchSuffix = $portVersion
+	lib:libblkmaker_jansson_0.1$secondaryArchSuffix = $portVersion
+"
+
+PROVIDES_devel="
+	devel:libblkmaker_0.1$secondaryArchSuffix = $portVersion
+	devel:libblkmaker_jansson_0.1$secondaryArchSuffix = $portVersion
+"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	libbase58$secondaryArchSuffix
+	jansson$secondaryArchSuffix
+"
+
+REQUIRES_devel="
+	libblkmaker$secondaryArchSuffix == $portVersion
+"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	pkgconfig
```

Comment:
pkgconfig should be replaced by
```
	cmd:pkg_config$secondaryArchSuffix
```
and moved to BUILD_PREREQUIRES.
BTW, if you could also swap the `devel:libjanson` and `devel:libbase58` (in BUILD_REQUIRES) and sort alphapetically the cmd: lines of BUILD_PREREQUIRES that would be great.

Hmm, there is also an unneeded blank line at the end of BUILD_PREREQUIRES. Could you remove it, and insert a tab before the closing double-quotes? Thanks!